Application/Education/ClassRegister/Diary/Service/Entity/TblDiary.php

```diff
@@ -0,0 +1,240 @@
+     */
+    protected $Location;
+
+    /**
+     * @Column(type="bigint")
+     */
+    protected $serviceTblPerson;
+
+    /**
+     * @return bool|TblDivision
+     */
+    public function getServiceTblDivision()
+    {
+
+        if (null === $this->serviceTblDivision) {
+            return false;
+        } else {
+            return Division::useService()->getDivisionById($this->serviceTblDivision);
+        }
+    }
+
+    /**
+     * @param TblDivision|null $tblDivision
+     */
+    public function setServiceTblDivision(TblDivision $tblDivision = null)
+    {
+
+        $this->serviceTblDivision = (null === $tblDivision ? null : $tblDivision->getId());
+    }
+
+    /**
+     * @return string
+     */
+    public function getSubject()
+    {
+        return $this->Subject;
+    }
+
+    /**
+     * @param string $Subject
+     */
+    public function setSubject($Subject)
+    {
+        $this->Subject = $Subject;
+    }
+
+    /**
+     * @return string
+     */
+    public function getContent()
+    {
+        return $this->Content;
+    }
+
+    /**
+     * @param string $Content
+     */
+    public function setContent($Content)
+    {
+        $this->Content = $Content;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDate()
+    {
+
+        if (null === $this->Date) {
+            return false;
+        }
+        /** @var \DateTime $Date */
+        $Date = $this->Date;
+        if ($Date instanceof \DateTime) {
+            return $Date->format('d.m.Y');
+        } else {
+            return (string)$Date;
+        }
+    }
+
+    /**
+     * @param null|\DateTime $Date
+     */
+    public function setDate(\DateTime $Date = null)
+    {
+
+        $this->Date = $Date;
+    }
+
+    /**
+     * @return string
+     */
+    public function getLocation()
+    {
+        return $this->Location;
+    }
+
+    /**
+     * @param string $Location
+     */
+    public function setLocation($Location)
+    {
+        $this->Location = $Location;
+    }
+
+    /**
+     * @return bool|TblPerson
+     */
+    public function getServiceTblPerson()
+    {
+
+        if (null === $this->serviceTblPerson) {
+            return false;
+        } else {
+            return Person::useService()->getPersonById($this->serviceTblPerson);
+        }
+    }
+
+    /**
+     * @param TblPerson|null $tblPerson
+     */
+    public function setServiceTblPerson(TblPerson $tblPerson = null)
+    {
+
+        $this->serviceTblPerson = (null === $tblPerson ? null : $tblPerson->getId());
+    }
+
+    /**
+     * @return bool|TblGroup
+     */
+    public function getServiceTblGroup()
+    {
+
+        if(null === $this->serviceTblGroup){
+            return false;
+        } else {
+            return Group::useService()->getGroupById($this->serviceTblGroup);
+        }
+    }
+
+    /**
+     * @param null|TblGroup $serviceTblGroup
+     */
+    public function setServiceTblGroup(TblGroup $serviceTblGroup = null)
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$serviceTblGroup" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)